ReSwift/CoreTypes/Store.swift

```diff
@@ -28,7 +28,7 @@ open class Store<State: StateType>: StoreType {
             subscriptions.forEach {
                 // if a selector is available, subselect the relevant state
                 // otherwise pass the entire state to the subscriber
-                $0.subscriber?._newState(state: $0.selector?(state) ?? state)
+                $0.subscriber?._newState(state: $0.selector?(state) ?? state!)
```

Comment:
@tonyarnold @mjarvis Uh good catch! My memory is failing me and I can't find a reference as to why the reducer return type was changed to be optional. I'm pretty sure it wasn't intentional.

The good news is that we've been slow to ship a new release, so we could undo the change without breaking backwards compatibility.

I can't think of a case where it would be useful to have the entire app state be optional after `ReSwiftInit`.